shell/platform/linux/fl_renderer_gl.cc

```diff
@@ -61,6 +61,9 @@ static gboolean fl_renderer_gl_create_backing_store(
   backing_store_out->open_gl.framebuffer.user_data = provider;
   backing_store_out->open_gl.framebuffer.name = name;
   backing_store_out->open_gl.framebuffer.target = format;
+  backing_store_out->open_gl.framebuffer.destruction_callback = [](void* p) {
```

Comment:
This change seems unrelated to the description of the commit - what is the purpose of it?

Reply:
Leaving destruction_callback unset will result in crash mentioned in the issue.

Reply:
The commit message doesn't mention this - it says "Fix incorrect texture format with OpenGL ES". Should this be two PRs?

Reply:
This line is no longer need, since https://github.com/flutter/engine/pull/24845 is merged.

Reply:
Can you remove it from the PR then please, thanks.

Reply:
Done.